app/services/amr/csv_parser.rb

```diff
@@ -0,0 +1,45 @@
+require 'csv'
+
+module Amr
```

Comment:
Moved these in to an Amr name space as it's amr csv files which are being handled here